iosHackthonApp/Modules/Features/MealList/Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
```

Comment:
This is all we need to manage a module really. The app target includes MealList as a target that gets build when the app does, which means the app can use this as a framework (anything that is exposed publicly), as well as any dependencies that MealList includes.